packages/hardhat-core/src/internal/hardhat-network/stack-traces/solidity-stack-trace.ts

```diff
@@ -84,6 +86,13 @@ export interface PanicErrorStackTraceEntry {
   isInvalidOpcodeError: boolean;
 }
 
+export interface CustomErrorStackTraceEntry {
+  type: StackTraceEntryType.CUSTOM_ERROR;
+  message: string;
+  sourceReference: SourceReference;
+  isInvalidOpcodeError: boolean;
```

Comment:
I don't think this field is necessary. Aren't they always reverts?

Reply:
Yes, both for panic and custom errors. I removed it in aced929927af780b1413b2b73a437f956db3a9b4.